src/global.d.ts

```diff
@@ -8,10 +8,13 @@ declare interface Window {
         vm?: VM;
         blockly?: ScratchBlocks | null;
         loader?: ChibiLoader;
-        registeredExtension: Record<string, {
```

Comment:
unnecessary line breaks. please edit prettier config

Reply:
It is valid.

Reply:
it’s neither pretty nor does it improve readability much